.github/workflows/docker.yml

```diff
@@ -15,34 +15,58 @@
 name: Build and push Docker image to GCP Artifact Registry
 
 on:
-    workflow_dispatch:
-    push:
-      branches:
-        - main
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+  schedule:
+    # Every Monday at 1PM UTC (9AM EST)
+    - cron: "0 13 * * 1"
 
 jobs:
   build-and-push:
     runs-on: ubuntu-latest
 
     steps:
+      - name: Free Disk Space (Ubuntu)
+        uses: jlumbroso/free-disk-space@main
+        with:
+          # this might remove tools that are actually needed,
+          # if set to "true" but frees about 6 GB
+          tool-cache: false
+
+          android: true
+          dotnet: true
+          haskell: false
+          large-packages: false
+          docker-images: true
+          swap-storage: true
```

Comment:
You probably don't need any of these inputs - most of them are the defaults anyways (https://github.com/jlumbroso/free-disk-space/blob/main/action.yml) except for haskell and large-packages and I doubt we care about having haskell. The only input I'm not sure about is large-packages (not sure if that deletes anything we care about)

Reply:
well, I tried. Removing haskell and large-packages somehow has some strange errors: https://github.com/liferoad/dataflow-ml-starter/actions/runs/5592411027/jobs/10224828637. This is why I keep haskell and large-packages.

Reply:
Looks like that came from the large packages piece. So we should still be fine with:

```
      - name: Free Disk Space (Ubuntu)
        uses: jlumbroso/free-disk-space@main
        with:
          large-packages: false
      - name: Checkout
        uses: actions/checkout@v3
...
```

Reply:
verified it works by removing haskell (https://github.com/liferoad/dataflow-ml-starter/actions/runs/5601668367/jobs/10245871036). Thanks.